### PR TITLE
CA-186157: rrdp_iostat: Use new tapdisk3 stats struct

### PR DESCRIPTION
**Pull request by rafalmiel**

### CODE REVIEW

**Comment by xen-git** [comment]
Can one of the admins verify this patch for testing?

**Comment by johnelse** [comment]
@xen-git test this please

File: rrdp_iostat.ml [hunk 1]
```diff
@@ -313,42 +313,52 @@ module Blktap3_stats_wrapper = struct
...
+		) [] shm_vbds
+
+		(* List.fold_left (fun acc vbd ->
 			match read_raw_blktap3_stats vbd with
 			| Some stat ->
 				let domid, devid = Scanf.sscanf vbd "vbd3-%d-%d" (fun id devid -> (id, devid)) in
 				((domid, devid), stat) :: acc
 			| None -> acc
-		) [] shm_vbds
+		) [] shm_vbds *)
```
Comment (johnelse) [inline rrdp_iostat.ml]: If this code isn't needed any more then let's delete it :)

**Comment by johnelse** [comment]
...otherwise this looks fine so far, thanks!

File: rrdp_iostat.ml (hunk 1 above)
Comment (rafalmiel) [inline rrdp_iostat.ml]: I kept this, since similar logic would be needed when low_mem_mode info is added. Will make the change anyway, thanks

**Comment by rafalmiel** [comment]
I ammended the commit removing commented-out section

**Comment by johnelse** [comment]
Ok, looks good. I'll close this for now - feel free to reopen once the low_mem_mode part is ready.

**Pull request closed without merging**